### PR TITLE
Sc create activities

### PR DESCRIPTION
**Pull request by sallypikachu**
Added test

### CODE REVIEW

**Comment by davidrf** [comment]
@sallypikachu so far you have an index test. You also need a test for the show action, and another test for adding a new activity.

**Comment by sallypikachu** [comment]
Created a Activity model, controller, views along with tests. Merged master branch and fixed conflicts.

File: spec/features/02_user_sees_activitites_spec.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    expect(page).to have_content @bodaborg.description
+
+    click_link "All Activities"
+  end
+
+  scenario "add new activity" do
+    visit new_activity_path
+
+    fill_in "Activity Name", with: "Bouldering"
+    fill_in "Description", with: "Climb 25 feet high wall with only your bare hands"
```
Comment (radavis) [inline spec/features/02_user_sees_activitites_spec.rb]: Oh no! Four characters over the limit!

This is one of those cases where you are free to ignore the hound.

File: spec/features/02_user_sees_activitites_spec.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+# [] Visiting the `/activities` path should contain a list of restaurants.
+# [] Visiting the `/activities/10 path should show the details for a activity with the ID of 10.
+# [] Visiting the root path should display a list of all activities.
+
+feature "user sees a list of his/her activities" do
+  before(:each) do
+    @bodaborg = Activity.create(activity_name: "BodaBorg",    description: "20 challenging mental and physical courses", venue_id: 1, price_range: 2)
```
Comment (radavis) [inline spec/features/02_user_sees_activitites_spec.rb]: This line is kinda hard to read. Try this:

```
activity_attrs = {
  activity_name: "BodaBorg",    
  description: "20 challenging mental and physical courses", 
  price_range: 2
}

@bodaborg = Activity.create(activity_attrs)
```

You don't have a `Venue` model, yet. Don't worry about adding venue_id to your app until you have the model.

File: spec/features/02_user_sees_activitites_spec.rb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+    @skyzone = Activity.create(activity_name: "Trampoline Dodgeball", description: "Exactly as described", venue_id: 2, price_range: 1)
```
Comment (radavis) [inline spec/features/02_user_sees_activitites_spec.rb]: fix this line, and omit venue_id

File: spec/support/factory_girl.rb [hunk 4]
```diff
@@ -1,8 +1,23 @@
...
+    description {Faker::Hipster.paragraph}
+    sequence(:venue_id) { |n| "#{n}" }
+    price_range "2"
+  end
+
+  factory :review do
+    rating 5
+    review_body {Faker::Hipster.paragraph}
+    activity_id 1
+    user_id 1
```
Comment (radavis) [inline spec/support/factory_girl.rb]: There is no review model, yet. That means you don't need this factory, yet.

**Pull request merged**